### PR TITLE
Feat: support response http status code setting

### PR DESCRIPTION
**Pull request by jimmy319**
### Implementation note ###

To support response http status code setting, create a new attribute - `responseStatusCode` for `route` data model.

### CODE REVIEW

File: src/UniversalApp.js [hunk 1]
```diff
@@ -55,7 +56,7 @@ const initServer = function(app, routes, apolloServerOptions, production) {
                 : context
           })
             .then(html => {
-              res.status(200);
+              res.status(responseStatusCode);
```
Comment (mis101bird) [inline src/UniversalApp.js:59]: How about rename as `statusCode` ?

**Review by yulinscottkang — APPROVED** [review]
Cool, this is handy for error pages!

File: src/UniversalApp.js (hunk 1 above)
Comment (jimmy319) [reply src/UniversalApp.js:59]: Is `statusCode` clear enough? (The status code of routing?)

**Review by savemuse — APPROVED** [review]
👍

File: src/UniversalApp.js (hunk 1 above)
Comment (mis101bird) [reply src/UniversalApp.js:59]: I think it's clear enough.
What will be misunderstood with this?

Comment (jimmy319) [reply src/UniversalApp.js:59]: Actually I am not quite sure what does the status code of a route mean?

Comment (jimmy319) [reply src/UniversalApp.js:59]: Should a route be stateless?

Comment (mis101bird) [reply src/UniversalApp.js:59]: After searching other resource, I think you are right. `responseStatusCode` may be more specific!

**Review by mis101bird — APPROVED** [review]
LGTM

File: src/UniversalApp.js (hunk 1 above)
Comment (jimmy319) [reply src/UniversalApp.js:59]: Thanks for your feedback. Good discussion 👍

**Pull request merged**